### PR TITLE
docs: restructure of the navigation

### PR DESCRIPTION
**Pull request by burakukula**
**DO NOT MERGE**
**Umbrella PR for visual changes of the website**

The first step to the new approach to navigation.
In the next PR:
 - side nav adjustment in terms of styling
 - bringing back alphabetical order in the components sections - to be clarified with Dominik.

### CODE REVIEW

File: packages/components/workbench/README.mdx [hunk 1]
```diff
@@ -2,7 +2,6 @@
 title: 'Workbench'
 type: 'component'
 status: 'stable'
-section: 'advancedComponents'
```
Comment (burakukula) [inline packages/components/workbench/README.mdx:5]: I don't think we should have this section at all.

File: packages/website/components/Sidebar.tsx [hunk 2]
```diff
@@ -83,67 +81,101 @@ const components: Array<SidebarSectionType | SidebarLinkType> = [
...
+            <SidebarSection
+              title="Forma 36 version 3"
+              links={sidebarLinks.forma36Version3}
+              currentPage={currentPage}
+            />
+          </List>
+        </Flex>
+      );
+      break;
+  }
```
Comment (gui-santos) [inline packages/website/components/Sidebar.tsx]: I think we don't need a switch here
and we can avoid repeating some parts of the jsx if we do this:


```jsx
      return (
        <Flex
          as="nav"
          aria-label="Main Navigation"
          flexDirection="column"
          className={styles.nav}
        >
          <List className={styles.list}>
            {currentPage.includes('guidelines') && (
              <SidebarSection
                title="Guidelines"
                links={sidebarLinks.guidelines}
                currentPage={currentPage}
              />
            )}
            {currentPage.includes('components') && (
              <>
                <SidebarSection
                  title="Components"
                  links={components}
                  currentPage={currentPage}
                />
                <SidebarSection
                  title="Utils"
                  links={sidebarLinks.utils}
                  currentPage={currentPage}
                />
                <SidebarSection
                  title="Integrations"
                  links={sidebarLinks.integrations}
                  currentPage={currentPage}
                />
              </>
            )}
            ...
          </List>
        </Flex>
      );
```

File: packages/website/components/Topbar.tsx [hunk 3]
```diff
@@ -1,19 +1,23 @@
...
 
 const styles = {
   header: css`
-    display: flex;
-    background-color: ${tokens.blue700};
-    color: #fff;
+    display: grid;
+    grid-template-columns: 320px auto 320px;
+    column-gap: 0;
+    row-gap: 0;
```
Comment (gui-santos) [inline packages/website/components/Topbar.tsx:15]:
```suggestion
```

gap is 0 by default, no?

File: packages/website/components/Topbar.tsx [hunk 4]
```diff
@@ -1,19 +1,23 @@
...
+    background-color: ${tokens.colorWhite};
+    color: ${tokens.blue700};
     padding: 0 ${tokens.spacingXl};
     height: ${TopbarHeight};
-    align-items: center;
-    box-shadow: 0px 0px 10px 0px rgba(0, 0, 0, 0.2);
+    box-shadow: inset 0px -1px 0px #e7ebee;
```
Comment (gui-santos) [inline packages/website/components/Topbar.tsx:20]: inset box-shadow?

is that correct @domarku?
In the design I have it's a border

File: packages/website/components/Topbar.tsx [hunk 5]
```diff
@@ -60,84 +63,75 @@ const Logo = () => (
...
-    <circle fill="#ffffff" cx="80" cy="10" r="10" />
-    <circle fill="#ffffff" cx="10" cy="45" r="10" />
-    <circle fill="#ffffff" cx="45" cy="45" r="10" />
-    <circle fill="#ffffff" cx="10" cy="80" r="10" />
+    <circle fill={tokens.blue700} cx="45" cy="10" r="10" />
+    <circle fill={tokens.blue700} cx="10" cy="10" r="10" />
+    <circle fill={tokens.blue700} cx="80" cy="10" r="10" />
+    <circle fill={tokens.blue700} cx="10" cy="45" r="10" />
+    <circle fill={tokens.blue700} cx="45" cy="45" r="10" />
+    <circle fill={tokens.blue700} cx="10" cy="80" r="10" />
```
Comment (gui-santos) [inline packages/website/components/Topbar.tsx:71]: Did you try to pass fill to the `svg`?

I think it's possible and then we don't need to repeat the color several times

File: packages/website/components/SidebarLink.tsx [hunk 6]
```diff
@@ -10,13 +10,13 @@ export const getSectionTitleStyles = (isActive = false, indent = 1) => {
...
-      lineHeight: tokens.lineHeightL,
+      fontSize: tokens.fontSizeM,
+      lineHeight: tokens.lineHeightM,
     }),
     link: css({
       display: 'block',
-      fontSize: `${tokens.fontSizeL}`,
-      lineHeight: `${tokens.lineHeightL}`,
+      fontSize: `${tokens.fontSizeM}`,
+      lineHeight: `${tokens.lineHeightM}`,
```
Comment (gui-santos) [inline packages/website/components/SidebarLink.tsx:19]: Isn't it possible to use `<Text/>` for this?

**Review by gui-santos — CHANGES_REQUESTED** [review]
Looking good so far 🙌 

I have a few CSS & HTML suggestions 
I'm still checking it on the browser, so maybe I will add more comments sorry
I ❤️  CSS

File: packages/website/components/Sidebar.tsx (hunk 2 above)
Comment (burakukula) [reply packages/website/components/Sidebar.tsx]: Done!

File: packages/website/components/Topbar.tsx (hunk 4 above)
Comment (burakukula) [reply packages/website/components/Topbar.tsx:18]: hmm, I will double-check that, but I think I took it from Figma. thx

File: packages/website/components/Topbar.tsx (hunk 5 above)
Comment (burakukula) [reply packages/website/components/Topbar.tsx:69]: yeah, probably. I will handle it differently.

File: packages/website/components/Sidebar.tsx [hunk 7]
```diff
@@ -84,66 +82,69 @@ const components: Array<SidebarSectionType | SidebarLinkType> = [
...
+            >
+              Getting started
+            </SidebarLink>
+            <SidebarLink
+              isActive={isLinkActive('/contributing', currentPage)}
+              href="/contributing"
+            >
+              Contributing to Forma 36
+            </SidebarLink>
+            <Box marginBottom="spacingL" />
```
Comment (gui-santos) [inline packages/website/components/Sidebar.tsx:139]: Ah I almost forget

Could you remove all these `Box` used as spacing elements?

It's better to use CSS for this so we do not create empty HTML tags in the DOM

File: packages/website/components/SidebarLink.tsx (hunk 6 above)
Comment (burakukula) [reply packages/website/components/SidebarLink.tsx:19]: good point. I will try to do that. Thx!

File: packages/website/components/Topbar.tsx [hunk 8]
```diff
@@ -1,19 +1,23 @@
...
 
 export const TopbarHeight = '70px';
 
 const styles = {
   header: css`
-    display: flex;
-    background-color: ${tokens.blue700};
-    color: #fff;
+    display: grid;
+    grid-template-columns: 320px auto 320px;
```
Comment (gui-santos) [inline packages/website/components/Topbar.tsx:13]: uhmmmm I think we will need to update this once we merge the PR's updating the page's design
because these links are aligned to the content of the page, see:

Screenshot 2022-01-10 at 17 14 59

This basically means we will need to implement the same grid in the topbar and in the content

**Review by gui-santos — CHANGES_REQUESTED** [review]
I remembered a few more things, sorry

File: packages/website/components/Sidebar.tsx (hunk 7 above)
Comment (burakukula) [reply packages/website/components/Sidebar.tsx:139]: sure, I wanted to tacke it in the next PR. Side bar is not styles properly yet.

File: packages/website/components/Topbar.tsx (hunk 8 above)
Comment (burakukula) [reply packages/website/components/Topbar.tsx:13]: yeah. I think it's better to do it in a separate pr all together.

File: packages/website/components/SidebarLink.tsx (hunk 6 above)
Comment (burakukula) [reply packages/website/components/SidebarLink.tsx:19]: Let's follow up in the next PR. it will be subheading finally, so maybe let's do it in the next PR as soon as we have font sizes not overwritten.

File: packages/website/components/Topbar.tsx (hunk 4 above)
Comment (burakukula) [reply packages/website/components/Topbar.tsx:18]: we don't have this box shadow in our tokens. I will do the border for now. we will follow up with Dom.

**Review by suevalov — CHANGES_REQUESTED** [review]
I like the way it looks!

Few things that we need to do:

* Add `/playground` link to the top navigation
* `v3` at the right corner is confusing. I'd rather show the current version and add ability to switch to v3 (see chakra top bar as an example: https://chakra-ui.com/)

**Comment by burakukula** [comment]
@suevalov sure, we can rethink that. I can put playground to the top nav, although I don't think that’s really connected to how the nav is structured right now, it doesn’t seem logical to have a playground there, but sure, let's do it. 
v3 link to the right was actually discussed with Dominik, I think it might be a good idea to keep it there, it is not the most important thing that users should focus on, but it still has to be reachable (so I wouldn’t hide it in the footer).

**Review by suevalov — APPROVED** [review]

File: packages/website/components/Sidebar.tsx (hunk 7 above)
Comment (gui-santos) [reply packages/website/components/Sidebar.tsx:144]: Ah ok! that makes sense

File: packages/website/components/Footer.tsx [hunk 9]
```diff
@@ -25,47 +26,124 @@ export const Footer = () => {
...
-            Imprint / Legal
-          </TextLink>
-        </Flex>
+        <Flex flexDirection="column" marginRight="spacing2Xl">
+          <Flex marginRight="spacingL" marginBottom="spacingM">
+            <Link href="/playground" passHref>
+              <TextLink
+                href="/playground"
+                variant="secondary"
+                icon={<ExternalLinkIcon />}
```
Comment (gui-santos) [inline packages/website/components/Footer.tsx:35]:
```suggestion
```

This link is not opening a new tab so maybe no icon for it?

File: packages/website/components/Footer.tsx [hunk 10]
```diff
@@ -25,47 +26,124 @@ export const Footer = () => {
...
+          <Flex marginRight="spacingL" marginBottom="spacingM">
+            <TextLink
+              variant="secondary"
+              href="https://medium.com/contentful-design"
+              target="_blank"
+              rel="noopener noreferrer"
+              icon={<ExternalLinkIcon />}
+              alignIcon="end"
+            >
+              Contentful desing blog
```
Comment (gui-santos) [inline packages/website/components/Footer.tsx:64]:
```suggestion
              Contentful design blog
```

small typo

File: packages/website/components/Footer.tsx [hunk 11]
```diff
@@ -25,47 +26,124 @@ export const Footer = () => {
   return (
     <footer css={styles.footer}>
```
Comment (gui-santos) [inline packages/website/components/Footer.tsx:27]: The footer in the designs is white and has a border-top

but since you are not changing any CSS in this file, I wanted to ask if you are planing to do these changes in another PR... are you?

**Review by gui-santos — APPROVED** [review]
Amazing stuff! 🍰 

My only comment this time is a small typo

File: packages/website/components/Footer.tsx (hunk 11 above)
Comment (burakukula) [reply packages/website/components/Footer.tsx:27]: I think Dominik said something about keeping the gray, I was not sure if we want to do it. maybe let's follow up with him?

**Comment by suevalov** [comment]
A small thing: the footer seems too big for the playground page.

**Review by thebesson — APPROVED** [review]

**Review by suevalov — DISMISSED** [review]
To prevent merging
